README.md

```diff
@@ -1,7 +1,7 @@
 # vaadin-quarkus
 An extension to Quarkus to support Vaadin Flow.
 
-Supports Quarkus 3.12+
+Supports Quarkus 3.15+
```

Comment:
Could we also update this snippet:
```
* 1.0 for Vaadin 22 and 23 and Quarkus 2
* 2.0 for Vaadin 24 up to 24.4 and Quarkus 3.2 (LTS) and 3.8 (LTS)
```
to 
```
* 1.1 for Vaadin 23 and Quarkus 2
* 2.1 for Vaadin 24 and Quarkus 3.15 (LTS)
```

Reply:
That section refers to older Vaadin versions (`See other branches for other Vaadin versions:`)
So we should either remove the 2.0 line, since 24.4 is not supported anymore, or change the `This branch is compatible with upcoming Vaadin platform versions. See other branches for other Vaadin versions:` phrases to just mention the compatibility Matrix.
Or we can leave it as is, if we want to make it clear that Vaadin 24.0 to 24.4 requires version 2.0 of the extension

Reply:
I'd propose to remove 2.0 line (24.4 is not under support) and modify the snippet so that it describe the maintained branches:

```
This branch is compatible with upcoming Vaadin 24 platform versions and uses Quarkus 3.15 (LTS). See other branches for other Vaadin versions:
* 1.1 for Vaadin 23 and Quarkus 2
```

Reply:
done